docs/editor_guide.md

```diff
@@ -94,8 +93,69 @@ these steps need to be taken:
    ```
 2. set the GitHub topics, if they haven't been set yet. Topics should include at least `lesson` and the appropriate life-cycle badge.
 
-### After acceptance
+### Performing editorial checks
+Using the checklist above, perform the basic editorial checks to ensure that the
+lesson is ready for review.
+Share the results of the checklist with the authors in the thread,
+highlighting any points that need to be addressed before the lesson can be reviewed.
+
+When the editorial checks have been satisfied,
+you can begin looking for reviewers.
+
+### Finding reviewers
```

Comment:
what is considered a conflict for reviewers?

Reply:
Please refer to [our Conflict of Interest Policy](https://github.com/carpentries-lab/reviews/blob/main/docs/coi_policy.md)